sphinxcontrib/openapi/openapi30.py

```diff
@@ -164,6 +165,11 @@ def _example(media_type_objects, method=None, endpoint=None, status=None,
         except (ValueError, KeyError):
             status_text = '-'
 
+    # Provide request samples for GET requests
+    if method == 'GET' and not media_type_objects:
+        media_type_objects[''] = \
```

Comment:
style nit: The `\` isn't necessary here:

    media_type_objects[''] = {
        'examples': {'Example request': {'value': ''}}}

Reply:
Done.